src/Voron/StorageEnvironment.cs

```diff
@@ -90,6 +90,8 @@ public void ForceSyncDataFile()
         private readonly Logger _log;
         public static int MaxConcurrentFlushes = 10; // RavenDB-5221
 
+        internal readonly PrefetchedTables PrefetchedTables = new PrefetchedTables();
```

Comment:
This is globally accessed value that isn't safe to modify from multiple threads, but that is what you are doing there.